packages/api-explorer/src/components/Response/index.jsx

```diff
@@ -47,30 +85,36 @@ class Response extends React.Component {
         itemsResult.push({label: 'to examples', onClick: hideResults})
       }
     }
-    
-    const placeholderStyle = {
-      textAlign: 'center',
-      padding: '40px 0',
-      color: 'rgba(255,255,255,0.7)',
-      fontStyle: 'italic',
-      fontSize: '14px',
-    }
-   
+
+    console.log(result)
     return (
       <div>
-        {
-          result !== null ? (
-            <BlockWithTab
-              items={itemsResult}
-              selected={responseTab}
-              onClick={this.setTab} 
-            >
-              <div style={{maxHeight: '400px', padding: '10px', overflow: 'hidden scroll'}}>
-                {responseTab === 'result' && <ResponseBody result={result} oauth={oauth} isOauth={!!securities.OAuth2} />}
-                {responseTab === 'metadata' && <ResponseMetadata result={result} />}
+        {result !== null ? (
+          <BlockWithTab
+            items={itemsResult}
+            selected={responseTab}
+            onClick={this.setTab}
+          >
+            {!result.isBinary ? (
+              <div style={ctaContainerStyle}>
+                <CopyCode code={JSON.stringify(result.responseBody)} />
```

Comment:
With a json stringify, this responseBody must be a json. So why the check below?